.travis.yml

```diff
@@ -1,9 +1,11 @@
+dist: xenial
 language: python
 cache: pip
 python:
   - '3.4'
   - '3.5'
   - '3.6'
+  - '3.7'
```

Comment:
🎉

Reply:
Could probably even shove `3.8-dev` in here and make it optional for the moment 🤔

Reply:
Will do later; requires some additional changes to the YAML structure to get working